internal/mode/static/nginx/runtime/manager.go

```diff
@@ -27,6 +27,12 @@ type (
 	checkFileFunc func(string) (fs.FileInfo, error)
 )
 
+var (
+	noNewWorkersErr = "reload unsuccessful: no new NGINX worker processes started for config version %d." +
```

Comment:
```suggestion
	noNewWorkersErrFmt = "reload unsuccessful: no new NGINX worker processes started for config version %d." +
```

to make it clear it's a string format